schema/article.js

```diff
@@ -29,6 +29,13 @@ let ArticleType = new GraphQLObjectType({
     thumbnail_image: {
       type: Image.type,
       resolve: ({ thumbnail_image }) => thumbnail_image
+    },
+    slug: {
+      type: GraphQLString
+    },
```

Comment:
I guess for consistency maybe I could have called this `id` here, and just returned `slug`. Consistent w/ Gravity, which is the bulk of the endpoints.

Probably all schemas should have `cached`, `id` and `_id` and we can call it a convention.